### ISSUE TITLE
Ruby 3 should run the automated test suite

### ISSUE THREAD

**Issue by jaredcwhite** [bridgetownrb/bridgetown#173]
Using this issue to track problems/fixes when running `script/cibuild` under Ruby 3.

* Latest version of Listen gem doesn't support Ruby 3 yet.
* Rubocop v0.81 doesn't support Ruby 3…bumping up to v1.0 works but catches a ton of new offenses. (115 offenses detected, 78 offenses auto-correctable). We'll need to decide what's legit and what we need to reconfigure.
* Ruby Front Matter doesn't seem to be working (errors in `rendering.feature`)

**Comment by andrewmcodes** [comment]
@jaredcwhite I'd like to humbly suggest moving to standard and avoid the Rubocop config hell. Would also be less to maintain.

**Comment by jaredcwhite** [comment]
@andrewmcodes Ah hmm, huh, erm…how do I say this…I tried Standard and it just didn't work for me. I mean it worked, it just doesn't fit my brain. While I'm not gung-ho on Rubocop either (generally for my opinion on linters, I'll refer you to DHH 😉), I've managed to tweak the fairly reasonable config inherited from Jekyll/GitHub so that I'm fairly satisfied with it. I'm certainly open to suggestions if there are any rules that rub other folks the wrong way.

Maybe I should amend the above to say: "fix the Rubycop v1.0 config so it's in line with what we already have, unless there's an overarching reason why we should adopt a new rule/setting"
